UI/Web/src/theme/themes/dark.scss

```diff
@@ -57,365 +57,319 @@
+  --btn-primary-outline-text-color: white;
+  --btn-primary-outline-bg-color: transparent;
+  --btn-primary-outline-border-color: var(--primary-color);
+  --btn-primary-outline-hover-text-color: white;
+  --btn-primary-outline-hover-bg-color: var(--primary-color-darker-shade);
+  --btn-primary-outline-hover-border-color: var(--primary-color-darker-shade);
+  --btn-secondary-text-color: white;
+  --btn-secondary-bg-color: #6c757d;
+  --btn-secondary-border-color: #6c757d;
+  --btn-secondary-hover-bg-color: var(--bs-btn-hover-bg);
+  --btn-secondary-hover-border-color: var(--bs-btn-hover-border-color);
+  --btn-secondary-font-weight: bold;
+  --btn-secondary-outline-text-color: white;
+  --btn-secondary-outline-bg-color: transparent;
+  --btn-secondary-outline-border-color: transparent;
+  --btn-secondary-outline-hover-bg-color: transparent;
+  --btn-secondary-outline-hover-border-color: transparent;
+  --btn-secondary-outline-font-weight: bold;
+  --btn-primary-text-text-color: white;
+  --btn-secondary-text-text-color: lightgrey;
+  --btn-danger-text-text-color: var(--error-color);
+  --btn-alt-bg-color: #424c72;
+  --btn-alt-border-color: #444f75;
+  --btn-alt-hover-bg-color: #3b4466;
+  --btn-alt-focus-bg-color: #343c59;
+  --btn-alt-focus-boxshadow-color: rgb(255 255 255 / 50%);
+  --btn-fa-icon-color: white;
+  --btn-disabled-bg-color: #343a40;
+  --btn-disabled-text-color: white;
+  --btn-disabled-border-color: #6c757d;
+  --bs-btn-disabled-border-color: transparent;
+  /* Nav (Tabs) */
+  --nav-item-min-width: 150px;
+  --nav-item-max-width: 150px;
+  --nav-tab-border-color: rgba(44, 118, 88, 0.7);
+  --nav-tab-text-color: var(--elevation-layer10);
+  --nav-tab-bg-color: transparent;
+  --nav-tab-border-width: 0 0 3px 0;
+  --nav-tab-hover-border-color: var(--primary-color);
+  --nav-tab-active-text-color: white;
+  --nav-tab-border-hover-color: transparent;
+  --nav-tab-hover-text-color: white;
+  --nav-tab-hover-bg-color: transparent;
+  --nav-tab-border-top: transparent;
+  --nav-tab-border-left: transparent;
+  --nav-tab-border-bottom: var(--primary-color);
+  --nav-tab-border-right: transparent;
+  --nav-tab-hover-border-top: transparent;
+  --nav-tab-hover-border-left: transparent;
+  --nav-tab-hover-border-bottom: var(--primary-color);
+  --nav-tab-hover-border-right: transparent;
+  --nav-tab-active-hover-bg-color: transparent;
+  --nav-link-bg-color: var(--primary-color);
+  --nav-link-active-text-color: white;
+  --nav-link-text-color: var(--elevation-layer10);
+  /* Modal */
+  --modal-bg-color: #202122; // var(--bs-body-bg)
+  /* Header */
+  --nav-header-text-color: white;
+  --nav-header-bg-color: rgb(22, 27, 34);
+  /* Toasts */
+  --toast-success-bg-color: rgba(59, 158, 118, 0.9);
+  --toast-error-bg-color: #BD362F;
+  --toast-info-bg-color: #2F96B4;
+  --toast-warning-bg-color: #F89406;
+  /* Alerts */
+  --alert-text-color: #fff3cd;
+  --alert-bg-color: transparent;
+  /* Checkboxes/Switch */
```

Comment:
Please revert the indentation so it's easy to track changes. Bring the newlines back between the sections. If you're using auto formatters, please disable for this project.